recipe/meta.yaml

```diff
@@ -135,21 +133,15 @@ outputs:
     script: repack.bat  # [win]
     number: {{ mkl_buildnum|int + dstbuildnum|int }}
     requirements:
-      {% if win or 'conda-forge' not in my_channel_targets %}
-      host:
-        - intel-openmp {{ openmp_version.split('.')[0] }}.*
-        - tbb {{ tbb_version.split('.')[0] }}.*
-      run:
-        - intel-openmp {{ openmp_version.split('.')[0] }}.*
-        - tbb {{ tbb_version.split('.')[0] }}.*
-      {% else %}
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ c_stdlib }}_{{ target_platform }} >={{ c_stdlib_sycl_version }}  # [linux]
```

Comment:
Don't use `>=` at build time, but pin to a specific version that you need. This is because the version of `sysroot_linux` at build time will create a corresponding runtime requirement. For example, the current MKL 2025.0 builds are not installable on a linux system with `__glibc <2.28` anymore. Is that intentional?

Reply:
Yes, this is intentional for `onemkl-sycl-*` packages because sycl runtime/compiler requires glibc >= 2.28. Here it looks like I've accidentally set sycl like requirement on the glibc version